aimmo-game/tests/test_simulation/test_world_map.py

```diff
@@ -118,6 +120,22 @@ def test_all_cells(self):
         self.assertIn('D', cell_names)
         self.assertEqual(len(cell_names), 4)
 
+    def test_all_cells_python3(self):
+        # Except block will be called when python3 is used.
+        with self.assertRaises(Exception) as e:
```

Comment:
I think this test is wrong. This checks that an exception has been raised. The exception raised is in these lines:
```pyhton
self.assertFalse(isinstance(cell_names, collections.Iterable))
self.assertTrue(isinstance(cell_names, list))
``` 
since the assertion is false in python2. You can check this by commenting these lines. Thus, the test passes in python2. In python3, the test would fail since no exception would have been raised. 

I think we should have only one test, that should work regardless of the python version or no test at all.

Reply:
The previous test will pass on both versions of python; and I suppose test coverage isn't THAT important in this case. I'll get rid of it. Thanks! :+1:

Reply:
:+1:  I'm curious if that is something actually testable and how one would do that.